instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/aspects/TraceAspectAutoConfiguration.java

```diff
@@ -21,6 +21,7 @@
 @Configuration
 @EnableConfigurationProperties(TraceAspectProperties.class)
 @ConditionalOnProperty(prefix = "otel.springboot.aspects", name = "enabled", matchIfMissing = true)
+// TODO (trask) need to handle both WithSpan annotations here
```

Comment:
cc @mateuszrzeszutek

Reply:
Oh, right, I completely missed that -- I'll fix that in a separate PR, thanks for catching that!